.github/workflows/publish.yml

```diff
@@ -32,6 +32,8 @@ jobs:
 
       - uses: actions/setup-node@v4
         with:
+          node-version: 20
+          cache: 'npm'
```

Comment:
I vaguely remembering needing to keep cache off of the 2nd run. Its just used to trick the registry-url to change.

Reply:
I'll revert.

Reply:
fixed